ScoutSuite/output/data/html/partials/metadata.html

```diff
@@ -84,7 +84,7 @@
                   <li><a class="dropdown-item" href="javascript:showLastRunDetails()">Last run details</a></li>
                   <li><a class="dropdown-item" href="javascript:showAbout()">About Scout Suite</a></li>
                   <li><a class="dropdown-item" href="javascript:download_exceptions()">Export exceptions</a><a id="downloadAnchorElem" style="display:none"></a></li>
-                  <li><a class="dropdown-item">Dark theme
+                  <li><a class="dropdown-item" href="#">Dark theme
```

Comment:
Not quite sure about the href, since if the user missclicks while changing the theme he's sent to the Home page, but on the other hand it makes it homogeneous to the other drop-down items :man-shrugging:

Reply:
You're right, I didn't realize it would do that! I made a fix which keeps it homogeneous but doesn't redirect. It's a bit hacky but at least I don't have to add a css class specifically for this button.